### ISSUE TITLE
How to get fields from all users in a specific group?

### ISSUE THREAD

**Issue by phsmoura** [fedora-infra/fasjson#447]
Is it possible to get `username`, `human_name` and `website` fields from all users in a specific group?
This would be used in Fedora Planet to create the planet.ini file. The way it is now, the idea is to query members from the group and then query each user to get its info.

As we have +5k users in fedora-contributor, not all of them have `website` and it's needed to query 2 endpoints:
Would be possible to add an endpoint to get that info easier?
Or does an endpoint with this info already exist?
Or the idea of querying the group members first and then each user is really the one that should be used?

**Comment by abompard** [comment]
Yes, you can use the `X-Fields` header to select the fields you want:

```
curl -u : --negotiate \
  'https://fasjson.fedoraproject.org/v1/groups/fedora-contributor/members/?page_number=1' \
  -H 'X-Fields: username,human_name,website'
```

Would this work for you?

**Comment by phsmoura** [comment]
Thanks! Yes, that will work

**Comment by abompard** [comment]
Leaving the issue opened to add this example to the documentation

**Comment by inistack** [comment]
@abompard can this issue be assigned to me? I just wanted to let you know that I can get this done. 
I'm an Outreachy intern.

**Comment by abompard** [comment]
Sure, could you try assigning it to yourself using the link on the top of the right column? Thanks.

**Comment by inistack** [comment]
@abompard sorry, I currently do not have access to assign this to myself. Can you please help?

**Comment by TheQuietNote** [comment]
Hello i'm an outreachy intern and i would love to be assigned to this

**Comment by anshikavashistha** [comment]
I want to work on this @phsmoura .Please assign me

**Comment by Precious000** [comment]
Hi @abompard can you please assign this project to me? I’ll really appreciate, thanks!

**Comment by abompard** [comment]
I've assigned it to you @ginjardev , please tell me if you're no longer interested in this ticket.

**Comment by inistack** [comment]
I'm still interested and working on it. I'll send a PR soon.
